cmd/test/main.go

```diff
@@ -56,38 +67,56 @@ func NewTestCommand(ctx *TestCommandContext) *cobra.Command {
 }
 
 func test(ctx *TestCommandContext, paths []string, flags TestCommandFlags) error {
-	s := spinner.New(spinner.CharSets[9], 100*time.Millisecond)
-
-	s.Suffix = " Loading..."
-	s.Color("cyan")
+	s := createSpinner(" Loading...", "cyan")
 	s.Start()
 
-	messageChannel := bl.PopulateVersionMessageChan(ctx.VersionMessageClient, ctx.CliVersion)
+	messages := make(chan *messager.VersionMessage, 1)
+	go ctx.Messager.LoadVersionMessages(messages, ctx.CliVersion)
 
-	config, err := ctx.LocalConfig.GetConfiguration()
+	validPaths, invalidPaths, _ := ctx.Validator.Validate(paths)
+
+	evaluationId, err := ctx.Evaluator.CreateEvaluation(ctx.LocalConfig.CliId, ctx.CliVersion)
 	if err != nil {
 		fmt.Println(err.Error())
 		return err
 	}
 
-	evaluationResponse, fileParsingErrors, err := ctx.Evaluator.Evaluate(paths, config.CliId, 50, ctx.CliVersion)
+	results, errors, err := ctx.Evaluator.Evaluate(validPaths, invalidPaths, evaluationId)
```

Comment:
you chose here to forward the `invalidPaths` array into the evaluate function and let the evaluator deal with it. 
note that if we're giving the evaluator the 'power' to handle invalid files, we will need to later enhance it once we add the second validation - yaml validation (step 1).
not a deal breaker, just pointing that out.